core/domain/wipeout_service.py

```diff
@@ -207,7 +208,8 @@ def run_user_deletion(pending_deletion_request):
         return wipeout_domain.USER_DELETION_ALREADY_DONE
     else:
         delete_user(pending_deletion_request)
-        pending_deletion_request.deletion_complete = True
+        if auth_services.delete_user(pending_deletion_request.user_id):
+            pending_deletion_request.deletion_complete = True
```

Comment:
@vojtechjelinek I'm not sure this is the right thing to do, could you please give some guidance?

Reply:
I guess it is fine, I would maybe add logging message prefixed with `[WIPEOUT]` if the deletion fails.

Reply:
Also, please include the verification of the deletion, probably inside the `verify_user_deleted` funciton.

Reply:
And please add tests to `wipeout_service`, that will create a user with firebase account and then verify that everything was properly deleted.

Reply:
Hmm but now that I think about it, this will always fail until after the migration, because Firebase won't be able to initialize.

Should we wait for this PR to go in after migration?

Reply:
I guess it should happen in the same release, or it should be moved behind feature flag or some constant.

Reply:
Looks like it should be fine even without things happening in the same release, so marking as resolved.